docs/hugo/content/reference/appconfiguration/v1api20220501.md

```diff
@@ -1901,7 +1901,7 @@ identity and a set of user-assigned identities. The type &lsquo;None&rsquo; will
 <code>userAssignedIdentities</code><br/>
 <em>
 <a href="#appconfiguration.azure.com/v1api20220501.UserAssignedIdentityDetails_ARM">
-map[string]github/azure/azure-service-operator/v2/api/appconfiguration/v1api20220501.UserAssignedIdentityDetails_ARM
+map[string]./api/appconfiguration/v1api20220501.UserAssignedIdentityDetails_ARM
```

Comment:
Is this change correct? In some ways, the previous line looked more correct?

Reply:
I'll investigate